src/metabase/api/slack.clj

```diff
@@ -19,15 +19,18 @@
     (when (and slack-app-token (not config/is-test?))
       (when-not (slack/valid-token? slack-app-token)
         (throw (ex-info (tru "Invalid Slack token.")
-                        {:errors {:slack-app-token (tru "invalid token")}
-                         :status-code 400}))))
+                        {:errors {:slack-app-token (tru "invalid token")}}))))
```

Comment:
removed `:status-code` from the map since we add the 400 status in the `catch` block below